python/kserve/pyproject.toml

```diff
@@ -52,6 +52,8 @@ tabulate = "^0.9.0"
 pandas =  "^2.2.0"
 pydantic = [{ version = "^2.5.0", python = "3.12" }, { version = ">1.0,<3", python = ">=3.9,<3.12" }]
 pyyaml = "^6.0.0"
+msgspec = "~0.18.6"
+openai = ">=1.45.0"
```

Comment:
The original design is to not depend on the openai python library because we want to evolve the API in kserve.